projects/plugins/jetpack/tools/check-block-assets.php

```diff
@@ -75,7 +75,11 @@
 		'wp-element',
 		'wp-primitives',
 	),
-	'story'          => array(
+	'sharing-buttons' => array(
+		'react',
+		'react-dom',
+	),
```

Comment:
We shouldn't need this. Since we'll be adding such buttons to all posts on a site, we don't want to start loading React on every single post.

I think we would be better off loading individual svg files or an svg sprite, it would still be better for performance.